packages/web-components/src/switch/switch.ts

```diff
@@ -0,0 +1,21 @@
+import { attr, booleanConverter, FASTElement } from '@microsoft/fast-element';
+
+export class Switch extends FASTElement {
+  @attr value?: string;
+
+  @attr({ converter: booleanConverter }) isDisabled: boolean = false;
+
+  @attr({ converter: booleanConverter }) isChecked: boolean = false;
```

Comment:
As a convention, `is______` is somewhat redundant in my opinion. Since this is an attribute on the component, "is" essentially is implied. Were this a composed component or something more complex I think being specific on internals which are disabled would make sense. But as a convention on the component, `disabled` is a state and if it's boolean, then by being true it "is" disabled. :)